rom/scripts/GREEDY.md

```diff
@@ -0,0 +1,33 @@
+To use the greedy algorithm obtained from algorithm 2 of Choi et. al's
+paper "Gradient-based constrained optimization using a database
+of linear reduced-order models": https://arxiv.org/abs/1506.07849
+to build a ROM database with Laghos, use one of the following commands.
```

Comment:
Since greedy algorithm could mean other algorithms, and ROM databases could be constructed in many ways, perhaps elaborate some more here that this is for building an accurate ROM database for a finite simulation parameter space.